static/js/popovers.js

```diff
@@ -1139,6 +1139,9 @@ export function register_click_handlers() {
     });
 
     $("body").on("click", ".respond_personal_button, .compose_private_message", (e) => {
+        if (compose_state.composing()) {
+            compose_actions.cancel();
+        }
```

Comment:
Do these lines belong in `compose_actions.start` instead?  It seems like that would be a more complete fix not specific to this button.